infrastructure/platform/chronos.yaml

```diff
@@ -0,0 +1,75 @@
+---
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: chronos
+spec:
+  template:
+    spec:
+      containers:
+        - image: pocketsizefund/chronos:0.1.0
+          ports:
+            - containerPort: 8080
+          resources:
+            requests:
+              cpu: 100m
+              memory: 64Mi
+            limits:
+              cpu: 100m
+              memory: 128Mi
+---
+apiVersion: eventing.knative.dev/v1
+kind: Trigger
+metadata:
+  name: chronos-trigger
+spec:
+  broker: live
+  subscriber:
+    ref:
+      apiVersion: serving.knative.dev/v1
+      kind: Service
+      name: chronos
+---
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: morning-trigger
+spec:
+  schedule: "30 14 * * *"
```

Comment:
I'm confused by these two cron schedules - they seem backwards and outside of market hours (unless that's intended).